### ISSUE TITLE
Encoding finished by 101% or more

### ISSUE THREAD

**Issue by Enselic** [Enselic/recordmydesktop#27]
From https://sourceforge.net/p/recordmydesktop/bugs/79/

Sometimes, at the end of the encoding, the program shows a encoding process of over 100%, for example, the program shows me it finished the encodig by 104%. ;-)
This bug has been seen in recordMyDesktop 0.3.8.0 and gtk-recordMyDesktop 0.3.8. I didn't test other frontends, but they probably have this bug, too.

I have no idea what causes this bug nor I am able to reproduce this bug; to me the bug appears randomly. At most times, the bug does not apper and the proress correctly stops at 100%.

**Comment by juniorohanyere** [comment]
Yeah, I also noticed this, I even tried using CTRL+C to quit the process, nothing changed. It only continues on a new line. Until it had to reach 111%.
